Shared/sdk/SharedUtil.File.h

```diff
@@ -20,8 +20,8 @@ namespace SharedUtil
     //
     // Returns true if the file/directory exists
     //
-    bool FileExists(const SString& strFilename);
-    bool DirectoryExists(const SString& strPath);
+    bool FileExists(const SString& strFilename) noexcept;
+    bool DirectoryExists(const SString& strPath) noexcept;
```

Comment:
SString -> std::string

Reply:
Whats the point of SString if you dont use it anywhere?

SString inherits from std::string. Wouldnt make much difference

Reply:
We are trying to use only std::string in the new API. As far as I know, std::string -> SString casts have an allocation problem. So it's safe to use std::string here